### PR TITLE
Add torchvis support to dist bench

### PR DESCRIPTION
**Pull request by wconstab**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #89457
* #89330
* #89096
* #88523
* #89469
* __->__ #89324

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Review by davidberard98 — APPROVED** [review]

File: benchmarks/dynamo/distributed.py [hunk 1]
```diff
@@ -38,6 +49,7 @@ def run_model(args, model, inputs, key):
     setup(rank, world_size)
     if args.device == "cuda":
         # needed for FSDP
+        print(f"rank is {rank}, {type(rank)}")
```
Comment (albanD) [inline benchmarks/dynamo/distributed.py:52]: nit: maybe remove the print?

**Review by albanD — COMMENTED** [review]
Sounds good beyond extra print

**Review by albanD — APPROVED** [review]

File: benchmarks/dynamo/distributed.py (hunk 1 above)
Comment (wconstab) [reply benchmarks/dynamo/distributed.py:52]: thanks! i was being annoyed by that print and kept forgetting to fix it.

**Pull request closed without merging**